### ISSUE TITLE
Don't keep old develop version of plugins

### ISSUE THREAD

**Issue by ilija-lazoroski** [guardicore/monkey#3835]
## Describe the bug
If we missed the version of a plugin for whatever reason and push a plugin to the develop bucket and not decide to go on with that version then every other time we make changes to the plugin we will get the latest one.

## Expected behavior
We should not keep older plugin versions for development.

## Machine version (please complete the following information):
 - OS: Windows or Linux

**Comment by mssalvatore** [comment]
I don't understand what this is trying to say.

**Comment by ilija-lazoroski** [comment]
Somebody pushed a plugin with version 5.0.0. But we decided to not go on with that version so we left 4.0.0 on develop. After this everytime that we run a plugin from develop repository it will get the 5.0.0 and not the 4.0.0 which is an issue.

We talked on sync and decided to not really keep older versions of the plugins in the develop repo.

**Comment by mssalvatore** [comment]
Yeah, that should be the behavior, it's not sufficiently explicit in the design.

We shouldn't be upgrading major versions for any plugins at this point. All plugins have had their versions upgraded and have not yet been released.
